packages/vertix-base/src/modules/logger.ts

```diff
@@ -0,0 +1,201 @@
+import process from "process";
```

Comment:
/ask create a todo issue to move this file to another repo called @zenflux/logger